plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -880,6 +881,10 @@ public void createSchema(ConnectorSession session, String schemaName, Map<String
     public void dropSchema(ConnectorSession session, String schemaName, boolean cascade)
     {
         if (cascade) {
+            if (hasNestedNamespaces(session, schemaName)) {
+                throw new TrinoException(ICEBERG_CATALOG_ERROR, format("Cannot drop non-empty schema: %s", schemaName));
```

Comment:
This message should mention about nested namespaces.

Reply:
done